slapp/data_selection/segmentation_manifest.py

```diff
@@ -0,0 +1,102 @@
+import slapp.utils.query_utils as qu
```

Comment:
I'd note at the top that this will only work for LIMS.

Reply:
noted.